jujugui/static/gui/src/app/init/component-renderers-mixin.js

```diff
@@ -351,6 +353,31 @@ const ComponentRenderersMixin = (superclass) => class extends superclass {
   }
 
   /**
+    Opens the help overlay.
+  */
+  _renderHelp(state, next) {
+    ReactDOM.render(<Help
+      changeState={this.state.changeState.bind(this.state)}
+      displayShortcutsModal={this._displayShortcutsModal.bind(this)}
+      gisf={this.applicationConfig.gisf}
+      user={this.user}
+      webHandler={new WebHandler()}
```

Comment:
should probably just pass in the single `get` method

Reply:
Done